UofS-Pulse-Binfo/analyzedphenotypes
# Phenotype ontology

This PR will allow user to create/alter crop ontology term associated to a trait.

**Crop Ontology**
![Screen Shot 2019-07-03 at 2 23 57 PM (2)](https://user-images.githubusercontent.com/15472253/60622841-7e3f9300-9d9e-11e9-881f-c9563deade86.png)

## Metadata

 - Issue https://github.com/UofS-Pulse-Binfo/analyzedphenotypes/issues/98




Documentation:

## Description



## Dependencies



## Testing?


To test:

1. This field should be added to Trait content type. Click 'Check for new fields' in the manage fields page of Trait content type.
![Screen Shot 2019-07-03 at 2 34 22 PM (2)](https://user-images.githubusercontent.com/15472253/60623540-20ac4600-9da0-11e9-9463-251277d34905.png)

The field is added automatically in both Manage Field and Manage Display.
![Screen Shot 2019-07-03 at 2 39 04 PM](https://user-images.githubusercontent.com/15472253/60623625-5ea96a00-9da0-11e9-8249-e6db88faee7d.png)

2. Make sure to move this newly added field out of disabled field in Manage Display.
3. When instructed to set the # of values allowed, set this value to 1 (from Unlimited) and click save.
This step is required only when prompted.
4. With all configuration set, select a trait content type to view details. If a trait contained a previously set crop ontology, this field will show the information as shown below:
![Screen Shot 2019-07-03 at 2 23 57 PM (2)](https://user-images.githubusercontent.com/15472253/60622841-7e3f9300-9d9e-11e9-881f-c9563deade86.png)

Otherwise, user can assign (create relationship) a crop ontology by clicking the Edit
![Screen Shot 2019-07-03 at 2 45 49 PM (2)](https://user-images.githubusercontent.com/15472253/60624011-69b0ca00-9da1-11e9-9da0-08c435cf568e.png)

5. Click Update Term to change or add ontology term.
![Screen Shot 2019-07-03 at 2 47 31 PM](https://user-images.githubusercontent.com/15472253/60624121-ad0b3880-9da1-11e9-8ad7-5db11c738e6d.png)

NOTE: A link to vocabulary details is provided only when a term can be search in or supported by Vocab lookup module.
